src/amo/components/AddonMeta.js

```diff
@@ -16,7 +16,7 @@ export class AddonMetaBase extends React.Component {
     const { addon, i18n } = this.props;
     const averageDailyUsers = addon.average_daily_users;
     const averageRating = addon.ratings.average;
-    const addonReviewCount = addon.ratings.count;
+    const addonRatingCount = addon.ratings.count;
```

Comment:
Yes, that's true. Technically, a review is just a rating that also has text but this count could contain ratings without text.